app/models/link.rb

```diff
@@ -19,6 +19,11 @@ class Link < ApplicationRecord
       .where(service_interactions: { service_id: service })
   }
 
+  HTTP_OK_STATUS_CODE = 200
+
+  scope :currently_broken, -> { where.not(status: HTTP_OK_STATUS_CODE) }
```

Comment:
The code is run by the integration tests, so it seemed like overkill to write slow database-bound unit tests for something like this.

Additionally, unit testing a simple scope like this seems a bit like writing tests for ActiveRecord.  What do you think? I'm comfortable with the current level of coverage for this.